### PR TITLE
Qscrollarea widgetresizeable

### PR DESCRIPTION
**Pull request by bluebird75**
Finish the job started in #147 with simpler gui-less approach

### CODE REVIEW

**Comment by altendky** [comment]
Is the plan to allow PRs that piecemeal add the dozens of parameters that PyQt adds to the constructor for the properties?

**Comment by bluebird75** [comment]
There is a grand plan for adding all properties and signals to the constructor of all QObject derived classes. But this is a huge plan. In the meantime, a user suggested this valid improvement, I'm all for taking it in. More examples are always a good thing in any case.

**Review by altendky — CHANGES_REQUESTED** [review]
I think the application creation can be dropped once https://github.com/python-qt-tools/PyQt5-stubs/pull/166 goes in, then this looks good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 * [#183](https://github.com/python-qt-tools/PyQt5-stubs/pull/183) Add missing operations on QSize
 * [#189](https://github.com/python-qt-tools/PyQt5-stubs/pull/189) Fix QListWidget, QTreeWidget and QTableWidget so that their respective items are now optional in many places when used as argument or return value
 * [#165](https://github.com/python-qt-tools/PyQt5-stubs/pull/165) allow `None` as argument for `QLineEdit.setText()`
+* [#148](https://github.com/python-qt-tools/PyQt5-stubs/pull/148) add `widgetResizable` parameter to `QScrollArea.__init__()` and
+    `objectName` parameter to `QGroupBox.__init__()` .
```
Comment (altendky) [inline CHANGELOG.md]: I would tend to make all text, at least outside of Python code, use semantic newlines.

```suggestion
* [#148](https://github.com/python-qt-tools/PyQt5-stubs/pull/148) add `widgetResizable` parameter to `QScrollArea.__init__()` and `objectName` parameter to `QGroupBox.__init__()`.
```

**Review by altendky — APPROVED** [review]

**Pull request merged**